api/models/PackageVersion.js

```diff
@@ -187,7 +187,7 @@ module.exports = {
 
         var collaboratorsPromise = Collaborator.findAll({
           include: [
-            { model: PackageVersion, as: 'maintained_packages', required: true, where: conditions }
+            { model: PackageVersion, as: 'authored_packages', required: true, where: conditions }
```

Comment:
not related to the parsing but the relations were wrong `maintained_packages` got me the collaborators that were a maintainer, while `authored_packages` got me the collaborators that were just a contributors, which is what I wanted